src/repel_boxes.cpp

```diff
@@ -1103,6 +1105,12 @@ DataFrame repel_boxes2(
     } // loop through all text labels
   } // while any overlaps exist and we haven't reached max iterations
 
+  if (verbose) {
+    if (elapsed_time > max_time)  Rprintf("%.2fs elapsed during %d iterations before completing text repel\n", max_time/1e9, iter);
```

Comment:
@slowkow how easy would it be to change `any_overlaps` to `n_overlaps`? it might be nice to report in the `max_iter` or `max_time`-limited cases how many overlaps were still detected at the end